pyartcd/pyartcd/pipelines/check_bugs.py

```diff
@@ -184,9 +184,10 @@ async def _slack_report(self):
             for i in report[k]:
                 message += f'\n{i}'
 
-        self.logger.info('Sending notification to Slack')
-        self.logger.debug(message)
-        await self.slack_client.say(message)
+            self.logger.info('Sending notification to Slack')
```

Comment:
This will send one slack message per version, I don't think we want that

Reply:
I think it is great to have a dedicated thread for each issue. I have been confusing myself and others by having conversations about different topics in the same thread, and I think a separate thread for each version gets us closer to a one-topic-per-thread situation. 

Multiple issues per version is rather rare, so this lgtm :)